### PR TITLE
Set up includes for the Collections page

### PR DESCRIPTION
**Pull request by rediris**
Presumably?

Dependency of: https://github.com/gymnasium/openedx-theme/pull/58

See https://github.com/gymnasium/tracker/issues/140

### CODE REVIEW

File: collections/meta.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: meta
+permalink: /static/meta/collections-meta/
+page_title: "Collections | Gymnasium"
+og_title: "Collections"
+og_description: "Chocolate bar danish chocolate. Tootsie roll gummi bears tiramisu halvah."
+og_art: img/og/collections-og.png
```
Comment (rediris) [inline collections/meta.md]: This image does not exist.

File: collections/meta.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: meta
+permalink: /static/meta/collections-meta/
```
Comment (rediris) [inline collections/meta.md]: Question: should this be `/static/hub-pages/meta/collections-meta/`?
I wasn't sure if there's some pattern we have to follow here.

**Comment by andrewpmiller** [comment]
@rediris — using "resolves" will automatically mark that issue as done... not sure if that's what you intended... since there are likely tasks yet to be completed on gymnasium/tracker#140 (i.e. updating the configuration json to add that page to edx).

Also, definitely a good stab at making this work... I committed a couple of changes so that the meta include picks up the right values from the meta.md (the layout looks at the permalink to map values to the tags... it's an ugly hack that we employed in order to use a single meta layout despite using very inconsistent variable names everywhere. It is hideous, but it works.

Will review with @jgagne tomorrow before merging

File: collections/meta.md (hunk 2 above)
Comment (andrewpmiller) [reply collections/meta.md]: yes and no 😉
You get mad props for making this

File: collections/meta.md (hunk 1 above)
Comment (andrewpmiller) [reply collections/meta.md]: I adjusted the path so that when we do have the right image (or a placeholder) it'll be where it should be...

**Comment by rediris** [comment]
> @rediris — using "resolves" will automatically mark that issue as done... not sure if that's what you intended... since there are likely tasks yet to be completed on [gymnasium/tracker#140](https://github.com/gymnasium/tracker/issues/140) (i.e. updating the configuration json to add that page to edx).
> 
> Also, definitely a good stab at making this work... I committed a couple of changes so that the meta include picks up the right values from the meta.md (the layout looks at the permalink to map values to the tags... it's an ugly hack that we employed in order to use a single meta layout despite using very inconsistent variable names everywhere. It is hideous, but it works.
> 
> Will review with @jgagne tomorrow before merging

@andrewpmiller perfect, thanks!

File: _includes/take5/test-top.html [hunk 3]
```diff
@@ -1,106 +1,88 @@
...
-          if (t && /pre/i.test(t.nodeName)) {
-            var i = e[a.language] || a.language;
-            t.setAttribute("data-language", i)
-          }
-        })
-      }
-    }();
-  </script>
-  <title>Take 5 tutorials | Gymnasium</title>
+	<meta charset="utf-8">
```
Comment (rediris) [inline _includes/take5/test-top.html]: @jgagne The indentation got really weird in this one - maybe it flipped from spaces to tabs? I can take a look if you'd like.

Comment (jgagne) [reply _includes/take5/test-top.html]: Oops, sorry about that. Fix coming.

**Review by jgagne — APPROVED** [review]

**Comment by jgagne** [comment]
@rediris — Looking good. And thanks for pushing this forward!

**Review by andrewpmiller — APPROVED** [review]
👍

**Pull request merged**